server/app/helpers/wait_helper.rb

```diff
@@ -28,7 +28,7 @@ def wait(timeout: WAIT_TIMEOUT, interval: WAIT_INTERVAL, message: nil, &block)
       if value
         return value
       else
-        debug "wait... #{message}" if message
```

Comment:
Eh... somehow accidentially reverted this change. Must have still had it open in my editor.